src/search/search.js

```diff
@@ -63,12 +63,16 @@ export const search = new Svue({
     },
     start(prevUrls, results, params) {
       if (results == null) return 0;
-      return (prevUrls.length * params.perPage) + 1;
+      var startIndex = 1;
```

Comment:
I still haven't been able to reproduce this error, I would like to understand this better before pushing this code